### ISSUE TITLE
Tunic font /s'h/ escaping problem

### ISSUE THREAD

**Issue by ImprovingRigmarole** [dirdam/fonts#4]
Greetings !

First of all thanks a lot for this font, it's amazing, I'm working on a small project and it's very useful to me rn!

I've found a problem with a specific accidental combination which can't be escaped: from the readme : [the `mishear` example](https://github.com/dirdam/fonts/blob/8689d9a5658c5108462155d88d21df162f2dc9d4/tunic/README.md?plain=1#L93) gives the tip to split those, but is actually a very good example of this problem I'm having.

Here I've typed `s h sh ch sch scch` : which gives 
![image](https://github.com/dirdam/fonts/assets/17810364/947129e0-003f-4d65-a44e-28f5178eb0bb):
the first part `s h sh ch` gives as expected the `s` `h` `sh` `ch` glyphs.
But for the second part, if I want to write `s+h` without having them turn into `sh`, 
then adding `c` doesn't actually help because it gets linked as `s+ch`,
so I tried adding another `c` to try to avoid the `ch`, so `scch`.
In the end two consecutive `c` appear to leave the first empty `c` alone but creates a space and looks exactly like `s ch` (with maybe a wider space?).

I'm not sure if there's a ligature feature my text renderer doesn't support, 
or if it would be needed to have either
-  the consonant escape `c`, or
- the `ch` glyph

changed to avoid this specific case ?

Any insight on this problem ?

Thanks forward !

**Comment by ImprovingRigmarole** [comment]
ok so I just realized that since all consonant ligatures (except `ng` & `dj`) have `h` as second character, so the problem actually applies on all of those : if one needs them splitted and inserts `c`, they'll all turn into `ch`.

**Comment by ImprovingRigmarole** [comment]
I also found that the `c` empty glyph has a width of 600 (like all other consonants) to serve as support for all the vowels which have width 0.

I solved the problem by adding a width 0 blank glyph on an unused character (used `#`) to allow to split the ligature in any possible combination, as keeping `c` as support for vowel only syllables is still needed.

I can share my modified font if needed, just let me know !

Thanks again for the font!
